api/src/main/java/org/openmrs/module/diagnosis/DiagnosisService.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+
+package org.openmrs.module.diagnosis;
+
+import org.openmrs.Encounter;
+import org.openmrs.Obs;
+import org.openmrs.Patient;
+import org.openmrs.api.OpenmrsService;
+import org.openmrs.Diagnosis;
+
```

Comment:
The tests are failing because of failure importing Diagnosis. Does this class exist

Reply:
The class does not exist yet but is being created on a different ticket(TRUNK-5339). So the tests should pass once TRUNK-5339 has been merged in and this branch updated.